### PR TITLE
node: add sequelize + express + opentelemetry sample app

### PR DESCRIPTION
**Pull request by aabmass**
Depends on #26

Adds a sample app as an example for users. See `README.md` for details.

This sample lets you easily verify the OpenTelemetry integration is working. I will link to this sample in the documentation.

### CODE REVIEW

**Review by odeke-em — APPROVED** [review]
LGTM, thank you @aabmass!

**Comment by aabmass** [comment]
@odeke-em can you look at #26? This one should be merged after that one

**Comment by odeke-em** [comment]
I've merged #26, please rebase from master and when ready I shall merge this one too. Thank you @aabmass.

**Comment by aabmass** [comment]
Rebased :)

**Comment by aabmass** [comment]
Rebased :)

File: nodejs/sqlcommenter-nodejs/samples/express-opentelemetry/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# in a separate terminal
+$ curl http://localhost:8000
+{"todos":[{"id":11,"title":"Do dishes","description":null,"done":false,"createdAt":"2020-11-06T01:59:50.111Z","updatedAt":"2020-11-06T01:59:50.111Z"},{"id":12,"title":"Buy groceries","description":null,"done":false,"createdAt":"2020-11-06T01:59:50.111Z","updatedAt":"2020-11-06T01:59:50.111Z"},{"id":13,"title":"Do laundry","description":"Finish before Thursday!","done":false,"createdAt":"2020-11-06T01:59:50.111Z","updatedAt":"2020-11-06T01:59:50.111Z"},{"id":14,"title":"Clean room","description":null,"done":false,"createdAt":"2020-11-06T01:59:50.111Z","updatedAt":"2020-11-06T01:59:50.111Z"},{"id":15,"title":"Wash car","description":null,"done":false,"createdAt":"2020-11-06T01:59:50.112Z","updatedAt":"2020-11-06T01:59:50.112Z"}]}
+
+# Original terminal will output
+[server]: Server is running at https://localhost:8000
+Google Cloud Trace export
+Google Cloud Trace batch writing traces
+Google Cloud Teacesauthenticating
+Google Cloud Teacesgot authentication. Initializaing rpc client
```
Comment (odeke-em) [inline nodejs/sqlcommenter-nodejs/samples/express-opentelemetry/README.md:42]: What's "Google Cloud Teaces" is that output perhaps by something getting mangled up?

Comment (aabmass) [reply nodejs/sqlcommenter-nodejs/samples/express-opentelemetry/README.md:42]: Yes definitely mangled up, let me re-run it

**Comment by aabmass** [comment]
Should be good to go

**Comment by odeke-em** [comment]
Nice, thank you! Let me merge it.

**Pull request merged**